### ISSUE TITLE
How to access data behind "count" column ?

### ISSUE THREAD

**Issue by MichaelPinatton** [MaartenGr/BERTopic#516]
Hi Maarten,
Discovered BERTopic and liking it ! Thank you.

Once I generate the topics, we have the dataframe with Topic/Count/Name and I would like to access the data behing the Count column.

![bertopic1](https://user-images.githubusercontent.com/73756983/164910504-e19bc699-15b0-469b-a2bd-948b5d525bf6.png)

In my example I entered a list of 3809 keywords to get their topics. And the sum of Count column is 3809, so my guess is each keyword is attached to a topic.

**My end goal is to have :**
Col1 : Keyword --- Col2 : Topic
keyword 1 --- Topic 3
keyword 2 --- Topic 11
keyword 3 --- Topic 6
keyword 4 --- Topic 3
Etc

So i can attribute each keyword to its "category" or "topic".

Is it something doable ?

**Comment by MaartenGr** [comment]
Hi Michael, thank you for your kind words. 

Typically, you would enter a list of documents from which you would like to extract the topics. So it is interesting to see this being used to cluster keywords and get their related topics! 

Since we know that every single document contains only a single word, we can extract the counts as follows. Whenever you run `.fit_transform(docs)` you will get the `topics` variable as an output. That variable is a list of topics where each topic is related to each document. However, since each document contains only a single word, each topic is instead related to a single keyword. 

To then extract the frequencies, we can simply run the following:

```python
import pandas as pd
from bertopic import BERTopic

# Train model
topic_model = BERTopic()
topics, probs = topic_model.fit_transform(my_list_of_keywords)

# Extract frequencies
frequencies = pd.DataFrame({"Keyword": my_list_of_keywords,"Topic": topics})
```

In other words, all information behind the "count" column can be found in `topics`.

**Comment by MaartenGr** [comment]
Seeing as this was active a while ago, I am closing this for now. If you run into any issues, please let me know and I'll make sure to re-open it!
